pcdsdevices/elliptec.py

```diff
@@ -0,0 +1,62 @@
+"""
+Classes for ThorLabs Elliptec motors.
+"""
+
+
+from ophyd import Component as Cpt
+from ophyd import Device
+from ophyd.signal import EpicsSignal, EpicsSignalRO
+
+
+class ELL_Base(Device):
```

Comment:
The Python convention for class names from PEP-8 are CamelCase without underscores.

Reply:
Got it. I'll fix that here and elsewhere.